CMakeLists.txt

```diff
@@ -222,31 +222,33 @@ if(IS_ABSOLUTE "${CMAKE_INSTALL_INCLUDEDIR}")
 else()
     set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
 endif()
-if (LIBDE265_FOUND AND NOT WITH_LIBDE265_PLUGIN)
+if (LIBDE265_FOUND AND (NOT PLUGIN_LOADING_SUPPORTED_AND_ENABLED OR NOT WITH_LIBDE265_PLUGIN))
     list(APPEND REQUIRES_PRIVATE "libde265")
 endif()
-if (X265_FOUND AND NOT WITH_X265_PLUGIN)
+if (X265_FOUND AND NOT (NOT PLUGIN_LOADING_SUPPORTED_AND_ENABLED OR WITH_X265_PLUGIN))
```

Comment:
Looks like there is a NOT at the wrong place here.

Reply:
You're right. Copy-paste error. Will fix in a moment.